.github/workflows/pre-release.yml

```diff
@@ -18,9 +18,22 @@ jobs:
 
       - uses: actions/checkout@v3
       - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
+        run: mvn --batch-mode --update-snapshots install
```

Comment:
Consider using the `-ntp` option to avoid polluting CI logs with "downloading" logs.

Reply:
Thanks, done.